examples/simple_layer/main.zig

```diff
@@ -62,7 +62,7 @@ pub fn asyncMain() !void {
 
     // Start compiling. This uses the inferred shapes from the BufferStore.
     // The shape of the input tensor, we have to pass in manually.
-    var compilation = try asynk.asyncGeneric(zml.compileModel, .{ allocator, model_shapes, .forward, .{input_shape}, platform });
+    var compilation = try asynk.asyncc(zml.compileModel, .{ allocator, Layer.forward, model_shapes, .{input_shape}, platform });
```

Comment:
We also need to update the documentation. E.g. in "Writing your first model", the `simple_layer` code is used as example.

I can deploy the docs.

Reply:
agreed Corentin, can you do the search/replace on the full repo ?

Reply:
Sure, let me do that quickly

Reply:
Actually, we didn't update the documentation when we switched to `asyncGeneric`, so... it's already up-to-date 😂